### ISSUE TITLE
"begin to return" announced on vertical balloon movement when charges do not return

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#398]
From #385, for https://github.com/phetsims/QA/issues/112, @terracoda said

> @jessegreenberg, I think the fix for #389 has resolved the random utterance of the "begin to return" description when moving right, however, now I hear it when I do a vertical move (up or down) after jumping left away from the wall.
>This is easy to reproduce.

>To reproduce:
get a good charge going on balloon
jump to wall or near wall
jump to near wall or center
move up or move down, and you should hear the incorrect description, "Negative charges begin to return".

**Comment by jessegreenberg** [comment]
Reproduced in a11y-view.

**Comment by jessegreenberg** [comment]
Thanks for finding this @terracoda. I believe this should be fixed in the above commit, can you please verify?

**Comment by terracoda** [comment]
@jessegreenberg, I verified that this issue is fixed! Thanks! It's sounding really good!
I'll go ahead and close this issue.
